Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -1,13 +1,26 @@
 module Superenv
+  alias x11? x11
+
   # @private
   def self.bin
     (HOMEBREW_SHIMS_PATH/"linux/super").realpath
   end
 
+  def xorg_recursive_deps
+    if xorg_installed?
+      @xorg_deps ||= Formula["linuxbrew/xorg/xorg"].recursive_dependencies.map(&:to_formula)
+    else
+      []
+    end
+  rescue FormulaUnavailableError
+    []
+  end
+
   def homebrew_extra_paths
     paths = []
     binutils = Formula["binutils"]
     paths << binutils.opt_bin if binutils.installed?
+    paths += xorg_recursive_deps.map(&:opt_bin) if x11?
```

Comment:
Good idea. For that matter, it might make sense to do the same thing for the include/pkgconfig/etc. paths too.